src/hotkeys.js

```diff
@@ -76,7 +76,7 @@
           return false;
         }
 
-        return (element.contentEditable && element.contentEditable == 'true');
+        return (element.contentEditable && element.contentEditable == 'false');
```

Comment:
`contentEditable` is a string, not boolean (sounds not too good, but that's how browser returns this value). So yes, probably first clause `element.contentEditable` is extra.